resources/js/components/Board.js

```diff
@@ -100,13 +124,201 @@ export class Board extends React.Component {
       <td> { folder} {'>'}</td>
       );
   }
+  //_onHoldStart and _onHoldEnd borrowed from: https://www.youtube.com/watch?v=A95mIE2HdcY
```

Comment:
I feel like we don't really need this citation anymore - the code has changed enough that it's not copied verbatim, and it depends on React functions that the tutorial didn't cover. We can keep it in if you'd like, though